roles/redpanda_broker/templates/configs/tiered_storage.j2

```diff
@@ -1,11 +1,8 @@
 cluster:
-  cloud_storage_access_key: THISVALUENOTUSED
```

Comment:
TLDR: no, because they can pass it in through the cli

More detailed explanation:

users can pass in dicts through the command line to set these to whatever they want just like I do in the taskfile/readme

```            
CLI_ARGS: "--extra-vars redpanda='{\"cluster\":{\"cloud_storage_segment_max_upload_interval_sec\":\"{{.SEGMENT_UPLOAD_INTERVAL}}\"}}'"
```

Meanwhile, setting them to be conditionally included in the template with Jinja2 causes YAML issues with mapping (remember that giant thread about mapping errors?) even if there is absolutely zero whitespace because of the function that handles mashing templates together. 

If we swap over to JSON for the templates -- which I very much want to do -- we can handle conditional inclusion in maps much more gracefully without ansible/python mangling things and this problem goes away.